code/modules/admin/topic.dm

```diff
@@ -2015,15 +2015,15 @@
 				H.reagents.add_reagent("spaceacillin", 20)
 				logmsg = "a heal over time."
 			if("Permanent Regeneration")
-				H.dna.SetSEState(GLOB.regenerateblock, 1)
+				H.dna.SetDNAState(GLOB.regenerateblock, DNA_SE, TRUE)
 				genemutcheck(H, GLOB.regenerateblock,  null, MUTCHK_FORCED)
 				H.update_mutations()
 				H.gene_stability = 100
 				logmsg = "permanent regeneration."
 			if("Super Powers")
 				var/list/default_genes = list(GLOB.regenerateblock, GLOB.breathlessblock, GLOB.coldblock)
 				for(var/gene in default_genes)
-					H.dna.SetSEState(gene, 1)
+					H.dna.SetDNAState(gene, DNA_SE, TRUE)
```

Comment:
I think the ordering for these two here needs to be `block, TRUE, DNA_SE` ?

Reply:
They sure do x__x;